src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,141 @@
+:root {
+  --blue-accent: #00acdc;
+  --white: #fff;
+  --gray-color: #616070;
+  --dark-blue-color: #060b35;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  transition: transform 0.3s;
+
+  &__title {
+    color: var(--dark-blue-color);
+    width: 166px;
```

Comment:
```suggestion

```
Consider avoiding hardcoded values for width and height except for some special cases (images, decorative elements)